support-data-rest/src/main/java/org/uniprot/api/crossref/request/CrossRefSearchRequest.java

```diff
@@ -31,5 +31,5 @@
     private String fields;
 
     @Positive(message = "{search.positive}")
```

Comment:
Doubt: Should we allow -1 here for download of everything

Reply:
If user doesn't pass size param, it means download all.